internal/status/status.go

```diff
@@ -77,9 +77,9 @@ func joinArtists(artists []spotify.Artist) string {
 	return list
 }
 
-func showProgressBar(progress, duration int) string {
+func showProgressBar(progress int, duration *spotify.Duration) string {
 	const length = 16
-	bars := length * progress / duration
+	bars := length * progress / int(duration.Milliseconds())
```

Comment:
Do we necessarily need to cast to an `int` here? I think you're allowed to do `int / int64`, right?

Reply:
Nope! Compiler doesn't like that conversion.